static/panes/compiler.ts

```diff
@@ -3382,18 +3384,20 @@ export class Compiler extends MonacoPane<monaco.editor.IStandaloneCodeEditor, Co
             );
             const numericToolTip = this.getNumericToolTip(word);
             if (numericToolTip) {
-                this.decorations.numericToolTip = {
-                    range: range,
-                    options: {
-                        isWholeLine: false,
-                        hoverMessage: [
-                            {
-                                // We use double `` as numericToolTip may include a single ` character.
-                                value: '``' + numericToolTip + '``',
-                            },
-                        ],
+                this.decorations.numericToolTip = [
```

Comment:
It's getting late, but this has become an array? Was it always supposed to be and worked by accident? 😊

Reply:
This is part of the `type Decorations = Record<string, monaco.editor.IModelDeltaDecoration[]>;` above! (Making the values always arrays for type simplicity sake)